css/index.css

```diff
@@ -327,48 +327,89 @@ header nav ul li .register p {
   height: 1480px;
   color: #E0E0E0;
 }
-.trade .primary_header {
+.trade .trade_header {
   display: -webkit-box;
   display: -ms-flexbox;
   display: flex;
+  text-align: center;
+}
+.trade .trade_header p {
+  font-size: 40px;
+  font-weight: 600;
+}
+.trade .primary_header {
   -webkit-box-orient: vertical;
   -webkit-box-direction: normal;
       -ms-flex-direction: column;
           flex-direction: column;
-  -webkit-box-align: center;
-      -ms-flex-align: center;
-          align-items: center;
-  text-align: center;
   background-image: url("../../img/background/section-trade-XL.png");
-  height: 450px;
-  margin: 100px 0;
-}
-.trade .primary_header p {
-  font-size: 40px;
-  font-weight: 600;
+  height: 350px;
+  margin: 100px 0 0 0;
 }
 .trade .primary_header span {
-  margin-top: 40px;
+  margin-top: 30px;
   color: #c2c2c2;
 }
 .trade .calculator {
   position: relative;
-  bottom: 240px;
+  bottom: 140px;
   margin: 0 auto;
   height: 350px;
-  width: 70vw;
+  width: 70%;
   background-color: #fff;
+  color: #303030;
   border-radius: 20px;
 }
-.trade .secondary_header {
+.trade .calculator .calculator_in {
   display: -webkit-box;
   display: -ms-flexbox;
   display: flex;
-  -webkit-box-align: center;
-      -ms-flex-align: center;
-          align-items: center;
-  text-align: center;
-  margin: 100px 0;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: row;
+          flex-direction: row;
+  padding-top: 30px;
+  font-size: 20px;
+}
+.trade .calculator .calculator_in input {
+  margin: 30px 12%;
+  height: 2rem;
+  font-size: 18px;
+  border-top-style: none;
+  border-left-style: none;
+  border-right-style: none;
+  outline: none;
+}
+.trade .calculator input:valid ~ .input_text,
+.trade .calculator input:focus ~ .input_text {
+  opacity: 0;
+}
+.trade .calculator .input_text {
+  position: relative;
+  bottom: 60px;
+  margin: 0 12%;
+  -webkit-transition: 0.5s;
+  transition: 0.5s;
+}
+.trade .calculator .left_in {
+  width: 50%;
+}
+.trade .calculator .left_in .text_in {
+  width: 80%;
+}
+.trade .calculator button {
+  margin: 5px 5% 5px 10%;
+  border: none;
+  color: #E0E0E0;
+  width: 140px;
+}
+.trade .secondary_header {
+  -webkit-box-pack: center;
```

Comment:
remove this shit this is pollyfill. if you want to use it install plugin with it